app/views/organizations/adoptable_pets/index.html.erb

```diff
@@ -60,8 +60,7 @@
             image_options: {src: pet.images.first, url: adoptable_pet_path(pet)}
           ) do |c| %>
             <%= c.with_badge do %>
-              <%= render BadgeComponent.new("Adoption Pending", class: "badge bg-info", when: pet.has_adoption_pending?) %>
-              <%= render BadgeComponent.new("Adopted", class: "badge bg-warning", when: pet.is_adopted?) %>
+              <%= status_badge(pet) %>
```

Comment:
So, for this particular view, we only need to show the adoption pending badge for a pet. Otherwise we don't need to show a badge for in foster or open.